### ISSUE TITLE
selecting an environment and then removing that from the CSV / builder / whatever causes app to crash

### ISSUE THREAD

**Issue by MikeLeber117** [markitondemand/iOS-EnvironmentManager#14]
If you have an environment selected and you re create the environment manager removing that environment, it will cause an exception

```
    func baseUrl(for entry: Entry) -> URL {
        let selectedEnvironment = currentlySelectedEnvironmentFor(entry)
        return entry.baseUrl(for: selectedEnvironment)!
    }
```
is ran, the force unwrap is a problem. Possibly we should return an optional here.. or guard elsewhere
i.e. - `    func baseUrl(for entry: Entry) -> URL` could check if  their is matching environment for the entry, if not it should clear the default and return 0 element again.

**Comment by MikeLeber117** [comment]
fixed in #15

**Comment by MikeLeber117** [comment]
fixed. will be in the next minor / micro release (3.0.2 or something)
